native/src/widget/scrollable.rs

```diff
@@ -37,8 +35,9 @@ where
     scrollbar_width: u16,
     scrollbar_margin: u16,
     scroller_width: u16,
+    scroll_horizontal: Option<Horizontal>,
```

Comment:
It seems `Horizontal` includes the same fields of the vertical scrollbar as well. I propose renaming `Horizontal` to `Scrollbar` and use `width` instead of `height`:

```rust
/// Properties of a scrollbar within a [`Scrollable`].
#[derive(Debug)]
pub struct Scrollbar {
    width: u16,
    margin: u16,
    scroller_width: u16,
}
```

And then here:

```rust
vertical_scrollbar: Scrollbar,
horizontal_scrollbar: Option<Scrollbar>,
```

We can rename the current private `Scrollbar` type to something else.

Reply:
I think length is more confusing. In my mind, the length of a scrollbar is its longitude, while the width of a scrollbar is the cross-axis length.

Furthermore, we are already using width similarly in other instances like border width, stroke width, etc.